e2e/e2e_test.go

```diff
@@ -240,34 +262,37 @@ func Test_E2E_Token_Transfer(t *testing.T) {
 	}
 
 	// 8. Wait for validators to update DB state after Scheduled TX is mined
-	time.Sleep(20 * time.Second)
```

Comment:
I think we will need a more stable fix, as we already sync we could have a re-try in case of timeout.